charts/open-zaak/templates/deployment.yaml

```diff
@@ -29,6 +29,21 @@ spec:
       serviceAccountName: {{ include "open-zaak.serviceAccountName" . }}
       securityContext:
       {{- toYaml .Values.podSecurityContext | nindent 8 }}
+
+      {{- if and .Values.persistence.enabled .Values.initContainers.volumePerms }}
```

Comment:
I will document this explicitly in the OZ docs for the Helm installation strategy.